qctrlopencontrols/dynamic_decoupling_sequences/dynamic_decoupling_sequence.py

```diff
@@ -265,35 +265,32 @@ def export_to_file(
 
 
 def convert_dds_to_driven_control(
-    dynamic_decoupling_sequence: DynamicDecouplingSequence = None,
-    maximum_rabi_rate: float = 2 * np.pi,
-    maximum_detuning_rate: float = 2 * np.pi,
+    dynamic_decoupling_sequence: DynamicDecouplingSequence,
+    maximum_rabi_rate: float,
+    maximum_detuning_rate: float,
     minimum_segment_duration: float = 0.0,
-    **kwargs,
+    name=Optional[str],
```

Comment:
```suggestion
    name: Optional[str] = None,
```
sorry, missed this one.

Reply:
Thanks!
I thought `mypy` or `pylint` would complain it.

Reply:
Haha, I was surprised too, but then when I thought about it I realised that according to those there's nothing wrong: `name` is an untyped parameter for which you're assigning the value `Optional[str]` as the default.